src/site/content/en/blog/display-override/index.md

```diff
@@ -0,0 +1,135 @@
+        <td>
+          This mode is similar to <code>standalone</code>, but provides the
+          user a minimal set of UI elements for controlling navigation (such
+          as back and reload).
+        </td>
+      </tr>
+      <tr>
+        <td><code>browser</code></td>
+        <td>A standard browser experience.</td>
+      </tr>
+    </tbody>
+  </table>
+</div>
+
+These display modes follow a [well-defined fallback chain](https://w3c.github.io/manifest/#dfn-fallback-display-mode)
+(`"fullscreen"` → `"standalone"` → `"minimal-ui"` → `"browser"`). If a browser does not support a given
+mode, it falls back to the next display mode in the chain.
+
+## Shortcomings of the `display` property
+
+The problem with this hard-wired fallback chain approach is threefold:
+
+- A developer cannot request `"minimal-ui"` without being forced back into the `"browser"` display mode in case `"minimal-ui"` is not supported by a given browser.
+- Developers have no way of handling cross-browser differences, like if the browser includes or excludes a [back button](https://twitter.com/ChromiumDev/status/1012065260625383425/photo/1) in the window for `"standalone"` mode.
+- The current behavior makes it impossible to introduce new display
+modes in a backward compatible way, since explorations like tabbed application mode do not have a
+natural place in the fallback chain.
+
+## The `display_override` property
+
+{% Aside %}
+  The `display_override` property is meant to solve special corner cases. In almost all
+  circumstances the regular `display` property is what developers should reach out for.
+{% endAside %}
+
+These problems are solved by the `display_override` property, which the browser considers *before*
+the `display` property. Its value is a sequence of strings that are considered in-order, and the
+first supported display mode is applied. If none are supported, the browser falls back to evaluating
+the `display` field.
+
+In the example below, the display mode fallback chain would be as follows.
+(The details of `"window-control-overlay"` are out-of-scope for this article.)
+
+1. `"window-control-overlay"` (First, look at `display_override`.)
+1. `"minimal-ui"`
+1. `"standalone"` (When `display_override` is exhausted, evaluate `display`.)
+1. `"minimal-ui"` (Finally, use the `display` fallback chain.)
+1. `"browser"`
+
+```json
+{
+  "display_override": ["window-control-overlay", "minimal-ui"],
+  "display": "standalone",
+}
+```
+
+{% Aside %}
+  The browser will not consider `display_override` unless `display` is also present.
+{% endAside %}
+
+To remain backward compatible, any future display mode will only be acceptable as a value of
+`display_override`, but not `display`.
+Browsers that do not support `display_override` fall back to the `display` property and ignore
+`display_override` as an unknown Web App Manifest property.
+
+{% Aside %}
+  The `display_override` property is defined independent from its potential values.
```

Comment:
```suggestion
  The `display_override` property is defined independently from its potential values.
```

Reply:
I'm not sure what this means, much less why it's relevant to me as a web developer. Does this mean that if my browser finds a `display_override` value it doesn't recognize that it will fail gracefully and roll back to something it does recognize?

Maybe that's what needs to be said.

Reply:
It's like standardizing the size of an envelop. We just say the envelop is 10x20x0.1. This makes no statement about what contents the envelop is designed for. It could be sheets of paper, leaves, or feathers; we don't care at this stage.